<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CinemaBuildingRepertoirSchedule</class>
 <widget class="QDialog" name="CinemaBuildingRepertoirSchedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_seans_date">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>130</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Data seansu:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_otherday">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>160</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Inny dzień</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_client_account">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>140</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Konto klienta</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_give_ticket">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Wydaj bilet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_tomorrow">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>160</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Jutro</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_schedule">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>871</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_create_account">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>101</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Utwórz konto</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_menu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>891</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>322</x>
      <y>0</y>
      <width>31</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>521</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_building">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_user">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_modyfydata">
         <property name="text">
          <string>Modyfikuj dane</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_logout">
         <property name="text">
          <string>Wyloguj</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
